src/ImageSharp/Image/Image{TColor}.cs

```diff
@@ -63,6 +63,28 @@ public Image(Stream stream, Configuration configuration = null)
             this.Load(stream);
         }
 
+#if !NO_FILE_IO
```

Comment:
I wonder whether we should use netstandard names as directives e.g. 

```
#if  (!NETSTANDARD11 && !NETSTANDARD12) 
    // Do your thang!
#endif
```

My reasoning is that we know what API's are available for each standard.